core/repository/api/pom.xml

```diff
@@ -34,6 +34,16 @@
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-api</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.github.tomakehurst</groupId>
+			<artifactId>wiremock-jre8</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>ch.qos.logback</groupId>
+			<artifactId>logback-classic</artifactId>
+			<scope>test</scope>
```

Comment:
I didn't think we needed to add a logback dependency just to support a logback test config.

Reply:
How else would you expect the unit test runner to find a logger implementation?

Reply:
I found out why this isn't needed. The pom file in the core module already contains one.

Reply:
Huh, I missed that. Fair enough. It doesn't hurt to have it here, but you're correct that it's not necessary.